packages/ui/src/components/BufficornAttributesList.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <div class="stat-container" v-for="resource in stats" :key="resource.key">
     <img class="icon" :src="getAssetPath(resource.key)" alt="icon" />
-    <div class="stat">
+    <div
+      class="stat"
+      :style="{
+        backgroundImage: `linear-gradient(90deg, var(--primary-color-opacity-1) 0%, transparent ${data[
+          resource.key
+        ] * 100}%)`
+      }"
+    >
```

Comment:
What do you think about move the logic to create this gradient to a function to make the html more readable? To do so, we can add a computed property that calls that function and returns the gradient
```suggestion
    <div
      class="stat"
      :style="{
        backgroundImage: createResourceGradient(resource)
      }"
    >
    ...
    

    createResourceGradient(resource) {
      const resourcePercentage = this.data[resource.key] * 100 
      return `linear-gradient(90deg, var(--primary-color-opacity-1) 0%, transparent ${resourcePercentage}%)`
    }
    </script
```